app/views/forms/edit/abs/edit-absNationalReport.js

```diff
@@ -561,7 +561,9 @@ define(['app', 'underscore', 'views/forms/edit/edit' , 'views/forms/edit/documen
     //==================================
     function getAbsDocuments (government) {
         var natSchemas = appConfigService.nationalSchemas;
-        var q  = '(realm_ss:' + realm.value.toLowerCase() + ' ) AND NOT version_s:* AND government_s:'+ government.identifier + " AND schema_s:(" + natSchemas.join(' ') + ")";
+        var q  = '(realm_ss:' + solr.escape(realm.value.toLowerCase()) + 
+                 ' ) AND NOT version_s:* AND government_s:'+ solr.escape(government.identifier) + 
+                 ' AND schema_s:(' + solr.escape(natSchemas.join(' ')) + ')';
```

Comment:
escape each value then then join(). otherwise you may end up escaping `space`
```suggestion
                 ' AND schema_s:(' + _.map(natSchemas, solr.escape).join(' ') + ')';
```